go/cmd/deploy/main.go

```
@@ -282,17 +282,17 @@ func executeDeploy(ctx context.Context, opts DeployOptions) error {
 	}
 	ui.PrintSuccess(fmt.Sprintf("Deployment created: %s", deploymentId))
 
-	// Track final version for completion info
-	var finalVersion *ctrlv1.Version
+	// Track final deployment for completion info
+	var finalDeployment *ctrlv1.Deployment
 
 	// Handle deployment status changes
 	onStatusChange := func(event DeploymentStatusEvent) error {
 		switch event.CurrentStatus {
```

golangci-lint (2.2.2)

missing cases in switch of type ctrlv1.DeploymentStatus: ctrlv1.DeploymentStatus_DEPLOYMENT_STATUS_UNSPECIFIED, ctrlv1.DeploymentStatus_DEPLOYMENT_STATUS_PENDING, ctrlv1.DeploymentStatus_DEPLOYMENT_STATUS_BUILDING, ctrlv1.DeploymentStatus_DEPLOYMENT_STATUS_DEPLOYING, ctrlv1.DeploymentStatus_DEPLOYMENT_STATUS_NETWORK

(exhaustive)